reader_test.go

```diff
@@ -564,6 +566,35 @@ func TestMediaPlaylistWithOATCLSSCTE35Tag(t *testing.T) {
 	}
 }
 
+func TestMediaPlaylistWithEnvivioSCTE35Tag(t *testing.T) {
+	f, err := os.Open("sample-playlists/media-playlist-with-scte35-envivio.m3u8")
+	if err != nil {
+		t.Fatal(err)
+	}
+	p, _, err := DecodeFrom(bufio.NewReader(f), true)
+	if err != nil {
+		t.Fatal(err)
+	}
+	pp := p.(*MediaPlaylist)
+	for i := 0; i < int(pp.Count()); i++ {
+		segment := pp.Segments[i]
+		assert.NotNil(t, segment.SCTE)
+		assert.NotNil(t, segment.SCTE)
```

Comment:
Is checking this value twice intentional?

Reply:
LOL